Sources/MongoSwift/MongoClient.swift

```diff
@@ -241,25 +226,65 @@ public class MongoClient {
     }
 
     /**
-     * Get a list of databases.
+     * Run the `listDatabases` command.
      *
      * - Parameters:
-     *   - options: Optional `ListDatabasesOptions` to use when executing the command
+     *   - filter: Optional `Document` specifying a filter that the listed databases must pass. This filter can be based
```

Comment:
I added the clarification here since apparently the filter only applies to fields in the output, so `nameOnly` (which is set in the other two) restricts the filter to just "name" based filtration. Here, you can filter on anything.